src/main/java/step1/qna/domain/Answer.java

```diff
@@ -43,9 +45,11 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
-    public Answer setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
+    public DeleteHistory delete(final User loginUser) {
+        validateAnswerAuthentication(loginUser);
+        this.deleted = true;
+        return new DeleteHistory(ContentType.ANSWER, this.getId(), this.getWriter(),
+            LocalDateTime.now());
```

Comment:
```suggestion
        DeleteHistory deleteQuestionHistory = new DeleteHistory(
                ContentType.QUESTION,
                this.getId(),
                this.getWriter(),
                LocalDateTime.now()
        );
```
개인적으로는 요런 개행이 더 좋은 것 같더라구요. 모든 코드에 대해서 일관적으로 적용할 수도 있구요.
아니면, 필드들을 적당히 지역변수로 분리해서 하나의 라인으로 나타내는 것도 괜찮아 보이네요.